tests/cross_functional/stress/test_noobaa_under_stress.py

```diff
@@ -82,7 +113,16 @@ def test_noobaa_under_stress(
             # Iterate and stress the cluster with object upload
             # and other IO operations
             total_iterations = 1
-            for i in range(1, total_iterations + 1):
+            if test_type == "breakpoint":
+                # In case of breakpoint test we increment the multiplier first and
+                # then decrement the multiplier
+                iterator = list(range(1, total_iterations + 1)) + list(
+                    range(total_iterations - 1, -1, -1)
```

Comment:
This seems incorrect, it is returning [1, 2, 3, 4, 5, 4, 3, 2, 1, 0]
You may have to use list(range(1, total_iterations + 1)) + list(range(total_iterations - 1, 0, -1)). Please check